File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profWorkerInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profWorkerInfo extends Auditable {
    public WorkerInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }

    //    public int ? WorkerTypeId { get; set; }
    //    public virtual WorkerType WorkerType { get; set; }

    public workerTypeId: number;

    public name: string;

    public surName: string;

    public taxCode: string;

    public contractNumber: string;

    public monthlySalary: number;
   
    public startDate: Date;

    public endDate: Date;

    public status: string; 
        
}

File: Architecture.Core/Entities/ProfWorkerInfo.cs
using System;
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;

namespace Architecture.Core.Entities
{
    public class ProfWorkerInfo : Auditable
    {
        public int WorkerInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public int? WorkerTypeId { get; set; }
        public virtual WorkerType WorkerType { get; set; }
        public string Name { get; set; }
        public string SurName { get; set; }
        public string TaxCode { get; set; }
        public string ContractNumber { get; set; }
        public decimal MonthlySalary { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Active { get; set; }

    }
}
